wpilibc/build.gradle

```diff
@@ -243,6 +243,7 @@ model {
                             def arch = it.targetPlatform.architecture.name
                             if (arch == 'x86-64' || arch == 'x86') {
                                 dependsOn it.tasks.install
+                                commandLine it.tasks.install.runScriptFile.get().asFile.toString()
```

Comment:
Is this a relevant change?

Reply:
Not directly, but the task was broken without it. I can move it to a separate PR if you want.